package dqs;

option cc_generic_services = true;
option py_generic_services = true;

enum RpcStatus {
  // make rpc successfully
  kRpcOk = 0;
  kRpcError = 2;
}

message PutDelayMsgRequest {
  optional uint32 delay = 1;
  optional string queue = 2;
  optional bytes value = 3;
}

message PutDelayMsgResponse {
  optional RpcStatus status = 1;  
}

service Dqs {
  rpc PutDelayMsg(PutDelayMsgRequest) returns(PutDelayMsgResponse); 
}


